.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
in the future, you want to create a gitignore folder and add this file in there so that it doesn't get pushed into your projects

Reply:
Thanks. I will do.